main_test.go

```diff
@@ -97,6 +97,13 @@ func TestGoTorchCommand(t *testing.T) {
 	mockVisualizer.AssertExpectations(t)
 }
 
+func TestGoTorchCommand(t *testing.T) {
+	testGoTorchCommand(t, "http://localhost")
+
+	// Trailing slash in url should still work.
+	testGoTorchCommand(t, "http://localhost/")
```

Comment:
This is a great idea. Is it typical to write test cases which call other test cases?